src/features/XIT/REP/REP.ts

```diff
@@ -1,10 +1,22 @@
 import REP from '@src/features/XIT/REP/REP.vue';
+import { getEntityNaturalIdFromAddress } from '@src/infrastructure/prun-api/data/addresses';
+import { getParameterSites } from './entries';
 
 xit.add({
   command: ['REP', 'REPAIR', 'REPAIRS'],
   name: 'REPAIRS',
   description: 'Shows the materials to repair buildings.',
   optionalParameters: 'Planet Identifier(s)',
-  contextItems: () => [{ cmd: 'BRA' }],
+  contextItems: (parameters: string[]) => {
+    const cmds = [{ cmd: 'BRA' }];
+    //sites will return [] if there are no parameters other than the command
+    //All other parameters will make a context item and add it
+    const sites = computed(() => getParameterSites(parameters));
+    if (!sites || !sites.value) return cmds;
+    sites.value.forEach(site => {
+      cmds.push({ cmd: `BRA ${getEntityNaturalIdFromAddress(site.address)}` });
+    });
```

Comment:
```suggestion
    for (const site of sites.value) {
      cmds.push({ cmd: `BRA ${getEntityNaturalIdFromAddress(site.address)}` });
    }
```
Code style: don't use `forEach` where `for..of` will suffice